ui/webui/src/components/AnacondaAboutModal.jsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState, useEffect } from "react";
+import {
+    AboutModal,
+    Button,
+    Flex,
+    TextContent,
+    TextList,
+    TextListItem,
+} from "@patternfly/react-core";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons";
+import { getAnacondaVersion, getOsReleaseByKey } from "../helpers/product";
```

Comment:
Please use: import { read_os_release as readOsRelease } from "os-release.js";

This changed since you implemented it, please rebase you PR to origin/master and adjust to the changes.